tomviz/CropOperator.cxx

```diff
@@ -0,0 +1,127 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "CropOperator.h"
+
+#include "CropWidget.h"
+#include "vtkDataObject.h"
+#include "vtkExtractVOI.h"
+#include "vtkImageData.h"
+#include "vtkNew.h"
+
+namespace tomviz
+{
+
+CropOperator::CropOperator(const int *dataExtent, const double *dataOrigin,
+                           const double *dataSpacing, QObject* prnt)
+  : Superclass(prnt)
```

Comment:
Really minor, but this seems ambiguous and I would rather we just used the supershort 'p' for parent. It is common in many projects, and this makes me think of print, and wonder what you are printing.